vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -1045,5 +1066,4 @@ testPamDSettings $OS_SKU $OS_VERSION
 testPam $OS_SKU $OS_VERSION
 testUmaskSettings
 testContainerImagePrefetchScript
-testNBCParserBinary
-testWasmRuntimesInstalled
+testNBCParserBinary
```

Comment:
suggest: need to add these two tests:
`testSpinKubeInstalled`
`testWasmRuntimesInstalled`

Reply:
Since they are called in the loop for components in components.json, I don't think they are needed here anymore.

Reply:
oh ok thanks for the clarification!